proxy/envoy/ingress.go

```diff
@@ -137,14 +147,27 @@ func generateIngress(mesh *proxyconfig.ProxyMeshConfig, tls *model.TLSSecret, ce
 	}
 
 	config := buildConfig(listeners, nil, mesh)
+
+	h := sha256.New()
+	hashed := false
```

Comment:
General question: should the cert/key writes here also be guarded with AuthPolicy==MUTAL_TLS check?